manifests/i/InfiniteRed/Reactotron/3.1.3.2591/InfiniteRed.Reactotron.installer.yaml

```diff
@@ -0,0 +1,30 @@
+# Created with YamlCreate.ps1 v2.2.12 $debug=QUSU.CRLF.7-4-0.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: InfiniteRed.Reactotron
+PackageVersion: 3.1.3.2591
+UpgradeBehavior: install
+ReleaseDate: 2023-12-20
+Installers:
+- Architecture: x64
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://github.com/infinitered/reactotron/releases/download/reactotron-app@3.1.2/Reactotron.3.1.2.msi
```

Comment:
Should this be 3.1.3 instead of 3.1.2? Same question for the other InstallerUrl.